### DISCUSSION TITLE
Leviathan - Crash when trying to complete Phase 5

### DISCUSSION THREAD

**Discussion by RedDeuces** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#32] [General]
I have finished everything and just need to turn in Phase 5 to complete the expedition. As soon as I click on the icon it crashes the game. Any idea what I can do to work around? It seems to work fine otherwise.

**Comment by RedDeuces** [comment]
Update: I checked my main save and says it is complete there and I was able to get the rewards. Just a strange bug.

**Comment by cwmonkey** [comment]
> Update: I checked my main save and says it is complete there and I was able to get the rewards. Just a strange bug.

Yep, this was a workaround because that quest reward is bugged and crashes the game. Nothing we can do about that, so I moved the reward to the end of phase 5 and made phase 5 optional so you can just end the expedition at the terminal when you're all done.

This info should probably be added to the readme @leodium

**Comment by ravl0** [comment]
@cwmonkey so what you mean, we can't finish the expedition and get the leviathan frigate anymore? because I finished all phases, and I have the same issue that I can't collect phase 5 and therefore I don't have the finish expedition button. All I can do is jump to main save but the rewards there are just cometics..
Also I can't transfer back the items I put from main save at the beginning of the expedition, but I guess I could live with that, but not getting leviathan? I think that is something worth mentioning in the readme files, I just wasted a few hours for nothing...
![1](https://github.com/user-attachments/assets/1dfce279-f635-4d67-a579-43822fbb0e88)

![2](https://github.com/user-attachments/assets/0cb008df-3da5-4913-b4c7-f9462af85d23)

**Reply by cwmonkey** [reply]
@ravl0 Go online, start the game and load your save. That will end the expedition and you will then be able to collect the frigate at the QuickSilver vendor.

**Comment by cwmonkey** [comment]
The frigate reward now seems to work properly. Latest cache json file should work (for now).
